tools/db_bench_tool.cc

```diff
@@ -2183,6 +2185,7 @@ class Benchmark {
 
     auto compression = CompressionTypeToString(FLAGS_compression_type_e);
     fprintf(stdout, "Compression: %s\n", compression.c_str());
+    fprintf(stdout, "Compression sampling rate: %ld\n", FLAGS_sample_for_compression);
```

Comment:
We use `fprintf(stdout, "Compression sampling rate: %" PRId64 "\n", FLAGS_sample_for_compression)` or `fprintf(stdout, "Compression sampling rate: %" PRIu64 "\n", FLAGS_sample_for_compression)`.

Reply:
Fixed.